python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -26,6 +25,7 @@
 from Ganga.GPIDev.Schema import ComponentItem, FileItem, GangaFileItem, Schema, SimpleItem, Version
 from Ganga.Runtime.spyware import ganga_job_submitted
 from Ganga.Utility.Config import ConfigError, getConfig
+from Ganga.Utility.external.OrderedDict import OrderedDict as oDict
```

Comment:
There was a convention but that started before there was any knowledge of an official ordered dictionary in Python. Since the official one was named `OrderedDict` and our own `OrderedDict.py` is named similarly and is designed to be compatible it makes sense to follow that convention.